docs/framework/winforms/controls/how-to-add-web-browser-capabilities-to-a-windows-forms-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Add web browser capabilities to app
+title: Add web browser capabilities to application
```

Comment:
I think app is better in this case since the title is so character restricted, it allows more of the suffix to show in search results if it's `app`.

```suggestion
title: Add web browser capabilities to app
```